### ISSUE TITLE
Slides to show with lazy load true

### ISSUE THREAD

**Issue by linxlad** [akiran/react-slick#788]
Firstly nice component :)

I am currently having an issue with slidesToShow > 1 and lazyLoad set to true.

On first load all the images are identical and it's not until the slider starts to cycle that the images then start changing due to the lazy load.

[https://jsfiddle.net/3ree9dqf/](https://jsfiddle.net/3ree9dqf/)

Any way around this issue?

**NB**: Worth a mention but setting `centerMode: true` doesn't seem to work?

Thanks

**Comment by ryanewing** [comment]
Did you find a workaround for this? I saw another issue that seemed to think it had to do with lazyLoading a react component, or one that doesn't re-pass props down to children.

**Comment by w3jimmy** [comment]
I did also update to the latest version, but still have the same issue.

**Comment by laveesingh** [comment]
I think most lazyloading related issues have been fixed. (latest release 0.19.0)
Please refer to this codesandbox example: https://codesandbox.io/s/n9p51n1lo0
If someone still has any problem regarding this, please generate a demo on codesandbox, and feel free to request reopen.
